Makefile

```diff
@@ -30,7 +30,7 @@ init-docs: startup
 
 docs:
 	if ! [ -d "./docs" ]; then poetry run sphinx-quickstart -q --ext-autodoc --ext-githubpages --ext-viewcode --sep --project $(PROJECT) --author '${AUTHOR}' docs; fi
-	cp ./pre_docs/* ./docs/source/
+	cp ./source_docs/* ./docs/source/
```

Comment:
no need to change to source_docs as it is equally confusing as pre_docs

the more important refactor is not to make sure `./docs` is not used as the place where docs-are-build (as now) and thus remains available for actually storing the docs -- what imho is the logical place (and not pre_ or source_) for people interacting with this project to put these ?

anyway -- not important enough to keep releases waiting...